src/test/test_http_client_builder.rs

```diff
@@ -0,0 +1,12 @@
+use reqwest::{
+    header::{self, HeaderMap, HeaderValue},
+    ClientBuilder,
+};
+
+/// Create a http client builder that with some default headers
+pub fn test_http_client() -> ClientBuilder {
```

Comment:
Can you add a unit test for this that checks that the header is set correctly when sending a request? And maybe a second test that confirms that it's still possible to add other headers as well?